### PR TITLE
Minor updates in enabling forging in Core docs v3 beta-4 - Closes #984

### LINKED ISSUE #984: Update and fix minor error in enable forging in Lisk core v3 beta-4
Minor update in enable forging page

### PR DESCRIPTION
**Pull request by chris529**
Closes #984

### CODE REVIEW

File: modules/ROOT/pages/management/forging.adoc [hunk 1]
```diff
@@ -147,26 +147,26 @@ If available, it is recommended to synchronize from snapshots to speed up the sy
...
+. Restore the `forger_info` table by executing the following command below:
 +
 [source,bash]
 ----
 lisk-core forger-info:import ./forger.db.tar.gz
 ----
 . <<adding-the-delegate-info-to-the-config>>.
 . Ensure the node is fully synchronized with the network.
-The height of your node should be equal to the current network height.
+The height of your node should be equal to the current network height. This can be achieved by executing the command below:
```
Comment (Tschakki) [inline modules/ROOT/pages/management/forging.adoc]: The sentence is confusing, as it is not achieved by the elow command. Please completely undo this change.

File: modules/ROOT/pages/management/forging.adoc [hunk 2]
```diff
@@ -147,26 +147,26 @@ If available, it is recommended to synchronize from snapshots to speed up the sy
...
-. Dump the data in the `forger_info` table of the db of your node.
+. Dump the data in the `forger_info` table of the db of your node by executing the following command below:
 +
 [source,bash]
 ----
 lisk-core forger-info:export
 ----
 . Login to the server with the new node.
-. Restore the `forger_info` table.
+. Restore the `forger_info` table by executing the following command below:
```
Comment (Tschakki) [inline modules/ROOT/pages/management/forging.adoc]: not necessary to add this here

File: modules/ROOT/pages/management/forging.adoc [hunk 3]
```diff
@@ -147,26 +147,26 @@ If available, it is recommended to synchronize from snapshots to speed up the sy
 . Login to the server with the old node.
 . <<forging-disable,Disable forging>> on the old node.
 . Stop the old node.
-. Dump the data in the `forger_info` table of the db of your node.
+. Dump the data in the `forger_info` table of the db of your node by executing the following command below:
```
Comment (Tschakki) [inline modules/ROOT/pages/management/forging.adoc]: not necessary to add this here

**Review by Tschakki — CHANGES_REQUESTED** [review]
Please see my comments and also solve the existing conflicts in the PR.

File: modules/ROOT/pages/management/forging.adoc (hunk 1 above)
Comment (chris529) [reply modules/ROOT/pages/management/forging.adoc]: OK, now done 👍

File: modules/ROOT/pages/management/forging.adoc (hunk 2 above)
Comment (chris529) [reply modules/ROOT/pages/management/forging.adoc]: OK, now done 👍

File: modules/ROOT/pages/management/forging.adoc (hunk 3 above)
Comment (chris529) [reply modules/ROOT/pages/management/forging.adoc]: OK, done 👍

**Review by Tschakki — APPROVED** [review]

**Pull request merged**